support-frontend/assets/pages/digital-subscription-landing/components/paymentSelection/helpers/paymentSelection.jsx

```diff
@@ -113,6 +117,47 @@ const BILLING_PERIOD_GIFT = {
   },
 };
 
+const getHeroCtaProps = (
+  productPrices: ProductPrices,
+  currencyId: IsoCurrency,
+  countryGroupId: CountryGroupId,
+): ProductSmall[] => {
+  const productOptions = getProductOptions(productPrices, countryGroupId);
+
+  const createPaymentOption = (billingPeriod: BillingPeriod): ProductSmall => {
+    const digitalBillingPeriod = billingPeriod === 'Monthly' || billingPeriod === 'Annual' ? billingPeriod : 'Monthly';
+    const productPrice = getProductPrice(productOptions, billingPeriod, currencyId);
+    const promotion = getAppliedPromo(productPrice.promotions);
+    const promoCode = promotion ? promotion.promoCode : null;
+    const offerCopy = promotion &&
+    promotion.landingPage &&
+    promotion.landingPage.roundel ? promotion.landingPage.roundel :
+      BILLING_PERIOD[digitalBillingPeriod].offer;
+    const trackingProperties = {
+      id: `subscribe_now__hero_cta-${billingPeriod}`,
```

Comment:
could we change this id to `subscribe_now_cta_hero-` to be consistent with `subscribe_now_cta-` and `subscribe_now_cta_gift-`?